rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java

```diff
@@ -296,6 +306,33 @@ public void onRemoval(
                 .build();
     }
 
+   public DataRead readData(String path, long readStart, int offset, int length)
+   {
+       DataRead dataRead = new DataRead();
+       byte[] buffer = new byte[CacheConfig.getBufferSize(conf)];
+       int nread;
+       BookKeeperFactory bookKeeperFactory = new BookKeeperFactory(this);
+       CachingNativeS3FileSystem fs = null;
+       try {
+           fs = new CachingNativeS3FileSystem(bookKeeperFactory, new Path(path), conf);
+           bufferSize = CacheConfig.getBufferSize(conf);
```

Comment:
Here, while creating an inputstream, we give a buffersize parameter. That will be the buffer  size in conf?